src/main/java/org/jenkinsci/plugins/github_branch_source/Connector.java

```diff
@@ -100,6 +100,7 @@ public class Connector {
     private static final String SALT = Long.toHexString(ENTROPY.nextLong());
     private static final OkHttpClient baseClient =
             JenkinsOkHttpClient.newClientBuilder(new OkHttpClient()).build();
+    private static final ThreadLocal<StandardCredentials> scanCredentials = new ThreadLocal<>();
```

Comment:
AFAICT this is now only ever set, and never read? And this is the only usage of `ThreadLocal` in this PR, so the title is now incorrect?

Reply:
right. I forgot to clean this one up. thanks!